packages/widget/src/components/transfer/fungible/fungible-token-transfer.ts

```diff
@@ -31,6 +31,9 @@ export class FungibleTokenTransfer extends BaseComponent {
   @property({ type: Object })
   onSourceNetworkSelected?: (domain: Domain) => void;
 
+  @state()
+  network: Domain | undefined;
```

Comment:
why do you need this? You have that in transferController

Reply:
to use it in updated lifecycle method

Reply:
Once you do https://github.com/sygmaprotocol/sygma-widget/pull/127#discussion_r1512402506 you won't need lifecycle method

Reply:
did it